### PR TITLE
Tabular

### PR DESCRIPTION
**Pull request by NSoiffer**
Change the examples in the alto use fake MathML with tables as suggested by @davidcarlisle in https://github.com/w3c/mathml/issues/354

The first example in #table-cells-that-are-not-divided-into-alignment-groups is not right though. The long first cell is spacing out the rest of the cells. I played with `table-layout` but didn't see a solution there. Maybe something with width and overflow might work. @davidcarlisle: do you know how to get this right?

### CODE REVIEW

**Review by davidcarlisle — CHANGES_REQUESTED** [review]
I pushed some general HTML validity commits, so approve modulo the (now) duplicated table example.

I added a second version with the long heading overflowing left and right

image


But I'm not sure that's the layout you intended, in either case, I left in the original table so one of them needs **removing** before this is merged.

The third table shown here with the short heading has a left aligned heading, which I'm not sure fits with the preceding description "`using the default columnalign="center"`" should that be centred? If it should be left aligned, presumably the long heading example should just overflow on the right, not left and right as I added here?

**Comment by NSoiffer** [comment]
Good catch. I was just trying replicate what was there, but what was there was wrong. Looking at the V3 spec, indeed the math should be centered. In the second example, the header should be centered.

I'm not thrilled with your solution of padding with `&nbsp;`, but I don't have a better idea. I'll adjust the examples.

**Comment by NSoiffer** [comment]
FYI: the short heading on the last example doesn't look centered, but it really is. Sort of an optical illusion.

**Pull request merged**
